src/CommandBinaryBuffer.ts

```diff
@@ -7,45 +7,37 @@ import * as deviceParameters from './constants/deviceParameters.js';
 
 export interface IBatteryVoltage {
     /**
-     * battery voltage value at low consumption, in mV;
+     * battery voltage value at low consumption, in mV
      *
-     * 4095 === undefined
+     * `4095` - unknown value
```

Comment:
with current implementation 4095 will be converted to 'undefined'
```suggestion
     * undefined - unknown value
```
?

Reply:
same in similar cases

Reply:
fixed